### DISCUSSION TITLE
Create community server on Discord?

### DISCUSSION THREAD

**Discussion by vanessa** [DjangoGirls/djangogirls#832] [Ideas]
Hi everyone,

I'm probably missing some context, but I remember Django Girls had a pretty active Slack server in the past. While Slack is useful for asynchronous communication and quick messages, it doesn't seem as friendly for community-building as Discord. That's why I was thinking it might be a good idea to create an open Discord server, mostly to improve communication and collaboration within the community, for both contributors and organizers.

In addition, here are some benefits of using Discord that we could take advantage of:

* Useful features for collaboration such as voice channels, screen sharing - these can be particularly helpful for clearing questions about contribution or pair programming in general.
* Easier for people to connect with each other, contribute, and meet other organizers, as they can quickly reach out to each other and share updates and ideas.
* Robust SDK so we can easily build a bot to automate tasks with slash commands (some of which are Django management commands in this codebase) and guide other people automatically if needed.
* We can host it for free on https://fly.io

Curious to hear your thoughts and ideas. If the team is interested, I'd be happy to help set up the server after it's created by someone from DG.

**Comment by ramonsaraiva** [comment]
I love Discord and have built different types of communities and applications in it. I share the same vision you have that DG's slack atm is quite calm, I believe Discord would bring us and the whole community closer.

Onboarding new people in Discord would probably be simpler as well. I've seen small communities become gigantic ones because of the features that come built-in a default Discord server - the public voice channels with screen sharing as you mentioned, is a big one.

The only downside, well, not really a downside but something to consider is that it will most likely require active moderation in public text and voice channels.

**Reply by vanessa** [reply]
@amakarudze what do you think?

**Reply by amakarudze** [reply]
I will have to check with the other trustees before I can give you any feedback on this. The requirement for active moderation may be a set back though.

**Reply by vanessa** [reply]
Happy to help with that if needed.

**Reply by vanessa** [reply]
Discord has some auto-moderation and permission settings that can definitely and easily control channel access, so it wouldn't require an active moderation team.

**Comment by vanessa** [comment]
@amakarudze what do you think if we take another look at this topic? I'm thinking it could be beneficial for the organizers - they aren't frequently active on this repository (since their main aim isn't contributing to the Django Girls website), it would enable them to promptly clarify any questions with the Django Girls team. And it could streamline their communication with other organizers/ambassadors/etc. and assist the Django Girls team with general operational tasks, using commands.

**Comment by marksweb** [comment]
I wonder if it might be possible to get a dedicated channel on the official django discord server?

**Reply by vanessa** [reply]
Absolutely, having a dedicated channel on the official Django Discord server could streamline things. We need to think about some potential issues here, though. If we only give moderation permissions to one or two individuals, we might accidentally create a bottleneck..

If Django Girls were to have its own Discord server, they could have more control and flexibility (e.g., they could create specific channels for different editions of Django Girls for direct contact between organizers and the Django Girls team). I believe this could really help make communication smoother and more efficient.

I'm definitely open to further discussion to find the solution that works best.

**Reply by marksweb** [reply]
I certainly see the benefit of running a server, I'm just conscious of the overhead that then exists in maintaining it.

I'm open to helping with this as well. I'm on the Django sever and part of the team running the django-cms servers for slack and discord.
